### DISCUSSION TITLE
Learning Pygame

### DISCUSSION THREAD

**Discussion by carolina-cza** [carolina-cza/Tecna#9] [Weekly Blog]
Hey hey!
Die letzte Woche habe ich mich hauptsächlich  mit pygame beschäftigt und mir da Sachen durchgelesen, ausprobiert, Visual Studio zum absturzt gebracht. Ich glaube der Fakt das ich  mein Visual Studio down gebracht hab, sag vieles über meine Testereien aus. Letzendlich weiß ich nicht mehr genau was der genau Grund für das down war, aber es funktioniert wieder alles einwandfrei.
Ein Tutorial von pygame was ich als sehr nützlich anseh ist  diese hier [ButtonTutorial](https://thepythoncode.com/article/make-a-button-using-pygame-in-python?utm_content=cmp-trueurl). Ich finde vor allem wen man Spiele oder ähnliches Programmiert braucht so gut wie immer ein Button, außer man macht alles über keyboard eingaben, aber so ein Button ist doch mehr dahinter als es zuerst aussieht. Ich finde auf der Seite wird es relativ gut erklärt wie er vorgeht und auch der code an sich ist sehr schlüssig aufgebaut, also könnt ihr es euch gern mal anschauen.

Ciao^^

**Comment by JonNoe** [comment]
Finde es gut, dass du dich mit pygame auch in deinem daily programming auseinandersetzt, so wird euer Gruppenprojekt auch durch die Implemention der Buttons gleich viel professioneller

**Comment by Amalia-Dann** [comment]
Hey naaa,
sehr cool dass du pygame ausprobiert hast. Aber wie hast du es geschafft VS zu abstürzen zu bringen??? Gutes Tutorial, das hab ich mir auch mal angeschaut für unser Spiel. 

Schauste mal, dass du mit deinem PRojekt nicht VS wieder zerstörst :)
Ansonsten viel Spaß <3
